src/test/blk_non_zero/TEST11

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019, IBM Corporation
```

Comment:
well, you copied an existing file and then modified it, so the original copyright should be maintained...

(please fix other files with this problem)